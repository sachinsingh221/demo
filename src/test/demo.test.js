import renderer from 'react-test-renderer';
import Link from '../components/component/Link';
import Demo from '../components/view/demo';
it('changes the class when clicked', () => {
  const component = renderer.create(
    <Demo></Demo>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
  renderer.act(() => {
    tree.props.handleModeChange();
  });
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});