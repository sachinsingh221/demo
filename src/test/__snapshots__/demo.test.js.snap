// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when clicked 1`] = `
Array [
  <div
    className="ant-radio-group ant-radio-group-outline"
    style={
      Object {
        "marginBottom": 8,
      }
    }
  >
    <label
      className="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
    >
      <span
        className="ant-radio-button ant-radio-button-checked"
        style={Object {}}
      >
        <input
          checked={true}
          className="ant-radio-button-input"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          type="radio"
          value="top"
        />
        <span
          className="ant-radio-button-inner"
        />
      </span>
      <span>
        Horizontal
      </span>
    </label>
    <label
      className="ant-radio-button-wrapper"
    >
      <span
        className="ant-radio-button"
        style={Object {}}
      >
        <input
          checked={false}
          className="ant-radio-button-input"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          type="radio"
          value="left"
        />
        <span
          className="ant-radio-button-inner"
        />
      </span>
      <span>
        Vertical
      </span>
    </label>
  </div>,
  <div
    className="ant-tabs ant-tabs-top"
    style={
      Object {
        "height": 200,
      }
    }
  >
    <div
      className="ant-tabs-nav"
      onKeyDown={[Function]}
      role="tablist"
    >
      <div
        className="ant-tabs-nav-wrap"
      >
        <div
          className="ant-tabs-nav-list"
          style={
            Object {
              "transform": "translate(0px, 0px)",
              "transition": undefined,
            }
          }
        >
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-0
            </div>
          </div>
          <div
            className="ant-tabs-tab ant-tabs-tab-active"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={true}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-1
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-2
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-3
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-4
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-5
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-6
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-7
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-8
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-9
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-10
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-11
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-12
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-13
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-14
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-15
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-16
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-17
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-18
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-19
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-20
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-21
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-22
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-23
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-24
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-25
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-26
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-27
            </div>
          </div>
          <div
            className="ant-tabs-tab ant-tabs-tab-disabled"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={true}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={null}
            >
              Tab-28
            </div>
          </div>
          <div
            className="ant-tabs-tab"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": undefined,
              }
            }
          >
            <div
              aria-controls={null}
              aria-disabled={false}
              aria-selected={false}
              className="ant-tabs-tab-btn"
              id={null}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="tab"
              tabIndex={0}
            >
              Tab-29
            </div>
          </div>
          <div
            className="ant-tabs-ink-bar ant-tabs-ink-bar-animated"
          />
        </div>
      </div>
      <div
        className="ant-tabs-nav-operations ant-tabs-nav-operations-hidden"
      >
        <button
          aria-controls="null-more-popup"
          aria-expanded={false}
          aria-haspopup="listbox"
          aria-hidden="true"
          className="ant-tabs-nav-more"
          id="null-more"
          onKeyDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          style={
            Object {
              "marginLeft": undefined,
              "order": 1,
              "visibility": "hidden",
            }
          }
          tabIndex={-1}
          type="button"
        >
          <span
            aria-label="ellipsis"
            className="anticon anticon-ellipsis"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="ellipsis"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
              />
            </svg>
          </span>
        </button>
      </div>
    </div>
    <div
      className="ant-tabs-content-holder"
    >
      <div
        className="ant-tabs-content ant-tabs-content-top"
        style={null}
      >
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={false}
          aria-labelledby={null}
          className="ant-tabs-tabpane ant-tabs-tabpane-active"
          id={null}
          role="tabpanel"
          style={Object {}}
          tabIndex={0}
        >
          Content of tab 
          1
        </div>
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
        <div
          aria-hidden={true}
          aria-labelledby={null}
          className="ant-tabs-tabpane"
          id={null}
          role="tabpanel"
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
        />
      </div>
    </div>
  </div>,
]
`;
